Game/Scripts/ComboManager.cpp

```diff
@@ -12,14 +12,14 @@ REGISTERCLASS(ComboManager);
 ComboManager::ComboManager() : Script(), 
 	input(nullptr), 
 	specialActivated(false), 
-	specialCount(0),
-	maxSpecialCount(100),
-	comboCount(0), 
-	maxComboCount(3),
+	specialCount(0.0f),
+	maxSpecialCount(100.0f),
+	comboCount(0.0f), 
+	maxComboCount(3.0f),
 	uiComboManager(nullptr),
 	comboManagerUIReference(nullptr),
-	comboTime(10),
-	actualComboTimer(0)
+	comboTime(10.0f),
+	actualComboTimer(0.0f)
 {
 	REGISTER_FIELD(comboManagerUIReference, GameObject*);
```

Comment:
We could change the type of this field.
```suggestion
	REGISTER_FIELD(uiComboManager, UIComboManager*);
```